jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties

```diff
@@ -1301,4 +1301,5 @@ K0808="Cannot create new file {0}"
 K0900="Create a new Reference, since a Reference cannot be cloned."
 
 #java.lang.invoke.MethodHandle
-K0A00="Failed to resolve Constant Dynamic entry with j9class: {0}, name: {1}, descriptor: {2}, bsmData: {3}"
\ No newline at end of file
+K0A00="Failed to resolve Constant Dynamic entry with j9class: {0}, name: {1}, descriptor: {2}, bsmData: {3}"
+K0A01="Constant_Dynamic references bootstrap method {0} does not have java.lang.invoke.MethodHandles.Lookup as first parameter."
```

Comment:
can you add quotes around the {0}?  ie: `'{0}'`

Reply:
quotes added